ote_cli/ote_cli/tools/utils/demo/visualization.py

```diff
@@ -145,7 +145,7 @@ def draw_predictions(task_type, predictions, frame, fit_to_size):
         frame = cv2.resize(frame, None, fx=ratio, fy=ratio)
     if task_type == TaskType.DETECTION:
         frame = draw_bounding_boxes(frame, predictions)
-    elif task_type == TaskType.CLASSIFICATION:
+    elif task_type in {TaskType.CLASSIFICATION, TaskType.ANOMALY_CLASSIFICATION}:
```

Comment:
would it be possible to show the heatmap here?

Reply:
maybe next time

Reply:
We will also have to enable heatmap from our side in the exportable code. I didn't add it as I thought it would confuse people.